inc/savedsearch_alert.class.php

```diff
@@ -393,12 +394,15 @@ static public function cronSavedSearchesAlerts($task) {
                   $alert->getFromDB($row['id']);
                   $data['savedsearch'] = $savedsearch;
                   NotificationEvent::raiseEvent($event, $alert, $data);
+                  $task->addVolume(1);
                }
             } catch (\Exception $e) {
                Toolbox::logDebug($e);
             }
          }
+         return 1;
```

Comment:
Maybe a real boolean s a better choice. Anyways, please update docbloc return tag to reflect the change

Reply:
Yes, I see... Well, not very important, boolean sounds more logic to me, but keeping 1/0 keeps consistency...
It's up to you. Just adapt the doc and it's ok ;)

Reply:
I have taken up in the doc what the Crontask::end($retcode) function expects as a return code that corresponds to the return of the cron function.